pkg/policyengine/policytypes/connection_request.go

```diff
@@ -0,0 +1,36 @@
+// Copyright 2023 The ClusterLink Authors.
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package policytypes
+
+type Direction int
+
+const (
+	Incoming Direction = iota
+	Outgoing
+)
+
+type ConnectionRequest struct {
+	SrcPeer string
```

Comment:
See earlier discussion regarding use of explicit `Source` and `Destination` structs to store the relevant attributes and making the attributes explicit in the communication between control plane and policy engine (instead of map[string]string as used for current state where attributes == Labels.